wgpu-rs/src/lib.rs

```diff
@@ -501,6 +503,10 @@ impl Device {
         let type_size = std::mem::size_of::<T>() as u32;
         assert_ne!(type_size, 0);
 
+        if count == 0 {
+            count = 1;
```

Comment:
we should be creating a buffer of length 1 if 0 is requested. I'm not seeing it in the code?

Reply:
Good point. It now creates them with size 1 instead of count 1.